### PR TITLE
Issue/3619

### LINKED ISSUE #3619: fix(admin-plugin): fix JS error in console when deactivating the Give Core plugin in 2.2.4
## User Story

As an `Admin` I should able to deactivate the Give Core plugin but currently, I am getting JS error in the console 

## Current Behavior

I currently getting JS error in the console 

## Expected Behavior

I expect no JS errro in console 

## Steps to Reproduce

1. Try to deactivate the Give Core 
2. check the console you will see a JS error 

## Visuals
image

## Environment

<details>
    <summary>WordPress System Info</summary>
### WordPress Environment ###

Home URL: http://givewp.local
Site URL: http://givewp.local
WP Version: 4.9.8
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
Show on Front: posts
Table Prefix Length: wp_
Table Prefix Length: 3
Table Prefix Status: Acceptable
Admin AJAX: Accessible
Registered Post Statuses: publish, future, draft, pending, private, trash, auto-draft, inherit, request-pending, request-confirmed, request-failed, request-completed, refunded, failed, revoked, cancelled, abandoned, processing, preapproval

### Server Environment ###

Hosting Provider: DBH: localhost, SRV: givewp.local
TLS Connection: Connection uses TLS 1.2
TLS Connection: Probably Okay
Server Info: nginx/1.10.1
PHP Version: 7.2.0
PHP Post Max Size: 1,000 MB
PHP Time Limit: 900
PHP Max Input Vars: 3000
PHP Max Upload Size: 1,000 MB
cURL Version: ❌ 7.38.0, OpenSSL/1.0.1t - We recommend a minimum cURL version of 7.40.
SUHOSIN Installed: –
MySQL Version: ❌ 5.5.55 - We recommend a minimum MySQL version of 5.6. See: WordPress Requirements
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
gzip: ✔
GD Graphics Library: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Give Configuration ###

Give Version: 2.2.3
Give Cache: Enabled
Database Updates: All DB Updates Completed.
Give Cache: Enabled
Give Cache: ✔New Donation✔Donation Receipt✔New Offline Donation✔Offline Donation Instructions✔New User Registration✔User Registration Information✔Email access
Upgraded From: 2.2.3
Test Mode: Enabled
Currency Code: USD
Currency Position: Before
Decimal Separator: .
Thousands Separator: ,
Success Page: http://givewp.local/donation-confirmation/
Failure Page: http://givewp.local/donation-failed/
Donation History Page: http://givewp.local/donation-history/
Give Forms Slug: /donations/
Enabled Payment Gateways: Test Donation, Offline Donation
Default Payment Gateway: Test Donation
PayPal IPN Verification: Enabled
PayPal IPN Notifications: N/A
Donor Email Access: Enabled

### Active Give Add-ons ###

Give - AWeber: ❌ Unlicensed – by WordImpress – 1.0.3

### Inactive Plugins ###

BuddyPress: by The BuddyPress Community – 3.0.0
Caldera Forms: by Caldera Labs – 1.5.9.1
Charitable: by WP Charitable – 1.6.1
Debug Bar: by wordpressdotorg – 0.9
Debug Bar Console: by koopersmith – 0.3
Debug Bar Cron: by Zack Tollman, Helen Hou-Sandi – 0.1.2
Debug Bar Extender: by Thorsten Ott, Automattic – 0.5
Developer: by Automattic – 1.2.6
Duplicate Post: by Enrico Battocchi – 3.2.2
Email Cop: by Ashfame – 0.1.1
Give - 2Checkout Gateway: by WordImpress – 1.1.1
Give - AmeriCloud Payments: by AmeriCloud Solutions, Inc. – 1.2.0
Give - Authorize.net Gateway: by WordImpress – 1.4.2
Give - Braintree Gateway: by WordImpress – 1.2
Give - CCAvenue Gateway: by WordImpress – 1.0.1
Give - ConvertKit: by WordImpress – 1.0.1
Give - Currency Switcher: by WordImpress – 1.2.2
Give - Display Donors: by WordImpress, LLC – 1.0
Give - Donation Upsells for WooCommerce: by WordImpress – 1.0.1
Give - Dwolla Gateway: by WordImpress – 1.1.2
Give - Email Reports: by WordImpress – 1.1.2
Give - Fee Recovery: by WordImpress – 1.6.1
Give - Form Countdown: by WordImpress – 1.0
Give - Form Field Manager: by WordImpress – 1.4.0
Give - Gift Aid: by WordImpress – 1.1.3
Give - GoCardless Gateway: by WordImpress – 1.2
Give - Google Analytics Donation Tracking: by WordImpress – 1.1.4
Give - MailChimp: by WordImpress – 1.4
Give - Manual Donations: by WordImpress – 1.4.1
Give - Mollie Gateway: by WordImpress – 1.1.2
Give - Paymill Gateway: by WordImpress – 1.1
Give - PayPal Pro Gateway: by WordImpress – 1.1.6
Give - Paytm Gateway: by WordImpress – 1.0
Give - PDF Receipts: by WordImpress – 2.3
Give - Razorpay: by WordImpress – 1.2.0
Give - Razorpay Recurring: by WP Buffs – 1.0
Give - Recurring Donations: by WordImpress – 1.7.2
Give - Stripe Gateway: by WordImpress – 2.0.6
Give - Tributes: by WordImpress – 1.4.2
Give - Zapier: by WordImpress – 1.1.2
Gravity Forms: by rocketgenius – 2.2.6
Gravity Forms Signature Add-On: by Rocketgenius – 3.4
Loco Translate: by Tim Whitlock – 2.1.4
Log Deprecated Notices: by Andrew Nacin – 0.4
Log Viewer: by Markus Fischbacher – 14.05.04
Monster Widget: by Automattic – 0.3
PHP Compatibility Checker: by WP Engine – 1.4.5
Polylang: by Frédéric Demarle – 2.3.6
qTranslate-X: by qTranslate Team – 3.4.6.8
Query Monitor: by John Blackbourn & contributors – 3.0.1
Rewrite Rules Inspector: by Daniel Bachhuber, Automattic – 1.2.1
User Switching: by John Blackbourn & contributors – 1.3.1
WooCommerce: by Automattic – 3.4.1
WordPress Beta Tester: by Peter Westwood – 1.2.2
WP Hide & Security Enhancer: by Nsp Code – 1.4.9.1
WPML Multilingual CMS: by OnTheGoSystems – 3.9.4
WPS Hide Login: by WPServeur, NicolasKulka, tabrisrp – 1.3.4.2

### Active MU Plugins ###

Local by Flywheel Relative URL (for Live Links): by Flywheel – 1.0

### Theme ###

Name: Twenty Seventeen
Version: 1.7
Author URL: https://wordpress.org/
Child Theme: No – If you're modifying Give on a parent theme you didn't build personally, then we recommend using a child theme. See: How to Create a Child Theme

</details>

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
PR to fix #3619 

## How Has This Been Tested?
Tested by deactivating the Give Core Plugin and also again byb activating 

## Screenshots (jpeg or gifs if applicable):
Video Link: Adding Video

image

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: assets/src/js/admin/plugin-deactivation-survey.js [hunk 1]
```diff
@@ -56,9 +56,9 @@ class GiveDeactivationSurvey {
 
 					// If no radio button is selected then throw error.
 					if ( 0 === checkedRadio.length && 0 === deactivationError.length ) {
-						surveyForm[ 0 ].innerHTML += `
+						surveyForm[0].innerHTML += `
 							<div class="notice notice-error deactivation-error">
-								${ Give.fn.getGlobalVar('deactivation_no_option_selected') }
+								${ Give.fn.getGlobalVar( 'deactivation_no_option_selected' ) }
```
Comment (ravinderk) [inline assets/src/js/admin/plugin-deactivation-survey.js]: @raftaar1191 did you forget to edit this?

File: includes/class-give-scripts.php [hunk 2]
```diff
@@ -193,8 +190,10 @@ private function plugin_equeue_scripts() {
 			'submit_and_deactivate'           => __( 'Submit and Deactivate', 'give' ),
 			'skip_and_deactivate'             => __( 'Skip & Deactivate', 'give' ),
 			'please_fill_field'               => __( 'Error: Please fill the field.', 'give' ),
+			'ajaxurl'                         => give_get_ajax_url(),
```
Comment (ravinderk) [inline includes/class-give-scripts.php]: @raftaar1191 remove this, WordPress provides `ajaxurl` in admin pages

File: assets/src/js/admin/plugin-deactivation-survey.js (hunk 1 above)
Comment (raftaar1191) [reply assets/src/js/admin/plugin-deactivation-survey.js]: Yup @ravinderk I remove it now

File: includes/class-give-scripts.php (hunk 2 above)
Comment (raftaar1191) [reply includes/class-give-scripts.php]: fix @ravinderk

**Comment by raftaar1191** [comment]
@ravinderk I have updated the PR as suggested by you

**Pull request merged**